src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -28,10 +32,10 @@ CUDNNEmitter::CUDNNEmitter(GPUPrimitiveEmitter* emitter)
 {
 }
 
-size_t CUDNNEmitter::build_reduce_forward(GPURuntimeContext* ctx,
-                                          const Shape& input_shape,
-                                          const AxisSet& reduction_axes,
-                                          const cudnnReduceTensorOp_t& reduce_op)
+size_t CUDNNEmitter::build_reduce_forward(const GPURuntimeContext* ctx,
+                                          const cudnnReduceTensorOp_t& reduce_op,
+                                          const ngraph::Shape& input_shape,
```

Comment:
If we are using namespace ngraph then we should not need `ngraph::`. You might need it because of the using ngraph::runtime::gpu if there is a Shape in that namespace, but just remove the multi level usings